src/JunctionTreeTypes.jl

```diff
@@ -200,8 +200,11 @@ mutable struct CliqStateMachineContainer{BTND, T <: AbstractDFG, InMemG <: InMem
   refactoring::Dict{Symbol, String}
   oldcliqdata::BTND
   logger::SimpleLogger
-  msgsUp::Vector{LikelihoodMessage} #TODO towards consolidated messages
-  msgsDown::Vector{LikelihoodMessage}
+  #TODO towards consolidated messages
+  # TBD DECISION, push or pull #674
+  # NOTE, DF going for Dict over Vector
+  msgsUp::Dict{Int, LikelihoodMessage} # Vector{LikelihoodMessage}
```

Comment:
dict because this is currently a "pull" model in the #674 sense of the word -- will probably change